Update batch-migration-of-legacy-public-folders.md (#2752)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md

- For information about keyboard shortcuts that may apply to the procedures in this topic, see [Keyboard shortcuts for the Exchange admin center](../../accessibility/keyboard-shortcuts-in-admin-center.md).

> - Please verify if the DefaultPublicFolderAgeLimit is configured on the organization level (Get-OrganizationConfig | fl DefaultPublicFolderAgeLimit) or if you have any AgeLimit (Get-PublicFolder <FolderPath> | fl AgeLimit) configured for the individual Public Folders, so that automatic deletions of the content to be prevented.

Reviewer: ```suggestion
- Verify if the **DefaultPublicFolderAgeLimit** parameter is configured on the organization level (Get-OrganizationConfig | fl DefaultPublicFolderAgeLimit) or if you have any age limit (Get-PublicFolder <FolderPath> | fl AgeLimit) configured for the individual public folders to prevent automatic deletion of the content.
```
